README.md

```diff
@@ -1 +1,95 @@
-# layout
\ No newline at end of file
+# Wolox Layout
+
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-layout.svg)](https://circleci.com/gh/Wolox/wolox-layout)
+[![npm](https://img.shields.io/npm/v/wolox-layout.svg)](https://www.npmjs.com/package/wolox-layout)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+
+## Features
+
+* `margins.scss` & `paddings.scss` files contains all the margin and padding classes you need to start a basic project.
```

Comment:
I'm not sure about the paddings stuff. I think paddings should be handled inside each component while margins alter the space separating an element and it's other siblings so makes more sense to keep them separated. My opinion is that I might want different separations (margins) for a component depending on where it is placed while paddings represent spacing inside a component and should be constant in every place. what do you guys think?